packages/openapi-to-graphql/src/oas_3_tools.ts

```diff
@@ -411,17 +411,9 @@ export function getSchemaType(schema: SchemaObject): string | null {
     // CASE: arbitrary JSON
     if (typeof schema.additionalProperties === 'object') {
       return 'json'
+    } else {
+      return 'object'
     }
-
-    // If there are no properties:
```

Comment:
I wanted to get rid of this. To return `null` in here is not only wrong but also ugly.